pkg/virt-controller/services/template.go

```diff
@@ -263,6 +263,10 @@ func (t *templateService) RenderMigrationManifest(vmi *v1.VirtualMachineInstance
 	if err != nil {
 		return nil, err
 	}
+
+	// migration target pods should connect to the same networks as the source and have the exact same network interfaces names
+	podManifest.Annotations[MultusNetworksAnnotation] = pod.Annotations[MultusNetworksAnnotation]
```

Comment:
There have been discussions offline about how to generate the target pod annotation, and this option did not pass.

We have no control of other webhooks or controllers that may react to the pod creation/change. Therefore, we are not to create a pod with annotations which have been added by other components. But this solution does exactly that.

I think we should just detect which interfaces names existed before, e.g. that in case there are secondary networks, they have `net<id>` format. In that case, to use that scheme. Otherwise, to use the new scheme.

Reply:
Done, the code was changed to generate the target pod annotation instead of using the existing migration source pod annotation.